CHANGELOG.asciidoc

```diff
@@ -22,6 +22,7 @@ https://github.com/elastic/apm-server/compare/71df0d96445df35afe27f38bcf734a0828
 - Use updated stack trace frame values for calculating error `grouping_keys` {pull}485[485] 
 
 ==== Added
+- Use index.auto_expand_replicas '0-1' by default {pull}503[503]
```

Comment:
nitpick: Should be at the bottom of the section

Reply:
🤦‍♂️